makePoplogCommander.sh

```diff
@@ -589,6 +593,14 @@ cat << \****
         } else if ( strcmp( "--help", argv[1] ) == 0 ) {
             printUsage( argc - 2, &argv[2] );
             return EXIT_SUCCESS;
+        } else if ( strcmp( "--version", argv[1] ) == 0 ) {
+****
+
+echo '            printf( "Poplog command tool v'${GET_POPLOG_VERSION:-Undefined}'\\n" );'
+echo '            printf( "Compiled against base Poplog '${POPLOG_VERSION:-Undefined}'\\n" );'
```

Comment:
Mmmm, I'm not sure about this one. What if I copy `poplog` into a new poplog tree with a different version, then the version string this prints will be wrong. Why don't we invoke `pop11` directly here to get the result? I don't see a good reason not to.

Reply:
Agreed. Fixed.